deploy.xml

```diff
@@ -20,8 +20,8 @@
 
     <licenses>
         <license>
-            <name>Commercial Vaadin Developer License version 4.0</name>
-            <url>https://vaadin.com/license/cvdl-4.0</url>
+            <name>Vaadin Commercial License</name>
```

Comment:
The full license name is "Vaadin Commercial License and Service Terms", I also updated the other PRs.

```suggestion
            <name>Vaadin Commercial License and Service Terms</name>
```